QUALITY_DECLARATION.md

```diff
@@ -125,19 +126,32 @@ Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linu
 
 ## Dependencies [5]
 
-`class_loader` has run-time and build-time dependencies that need to be considered for this declaration.
+Below are evaluations of each of `class_loader`'s run-time and build-time dependencies that have been determined to influence the quality.
+
+It has one "buildtool" dependency, which do not affect the resulting quality of the package, because they do not contribute to the public library API.
+It also has several test dependencies, which do not affect the resulting quality of the package, because they are only used to build and run the test code.
 
 ### Direct Runtime ROS Dependencies [5.i]/[5.ii]
 
-- console_bridge_vendor
-- rcpputils
+#### `console_bridge_vendor`
```

Comment:
@ahcorde meta: I'm a bit confused here, why do we depend on both the vendored package and the system dependency?

Reply:
These are the run dependencies that are in the `package.xml`. yo system may work with the vendored or the system dependency. The QD need to include both

Reply:
Right, the point being that the vendored package will (should?) take care of that fallback behavior, no?

Reply:
In other packages such as for example [rcl_yaml_param_parser](https://github.com/ros2/rcl/blob/master/rcl_yaml_param_parser/QUALITY_DECLARATION.md) we also define both dependencies

Reply:
You need to depend on both, if `console_bridge` [doesn't exist](https://github.com/ros2/console_bridge_vendor/blob/master/CMakeLists.txt#L72) then the vendor package will build it. I think this vendor package is only used for this. Then you need to depend on `console_bridge` to use the dependency.